apps/Flameshot/install-32

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 version=11.0.0
-install_packages https://github.com/flameshot-org/flameshot/releases/download/v${version}/flameshot-${version}-1.debian-10.armhf.deb || exit 1
+install_packages https://github.com/flameshot-org/flameshot/releases/download/v${version}/flameshot-${version}-1.debian-${get_codename}.armhf.deb || exit 1
```

Comment:
This does not appear to be tested.
First of all, incorrect syntax is used to call the `get_codename` function, and also, that function will return values like 'buster' or 'bullseye', NOT 10 or 11.
Also, this change will break Flameshot for our Ubuntu users.